### PR TITLE
Migrated test_mountpoint_ownership_post_volume_restart

### PR DESCRIPTION
**Pull request by aujjwal-redhat**
Migrated the [test
case](https://github.com/gluster/glusto-tests/blob/HEAD/tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py)

Updates: #292

Fixes: #581

Signed-off-by: Ayush Ujjwal <aujjwal@redhat.com>

### CODE REVIEW

File: common/ops/support_ops/io_ops.py [hunk 1]
```diff
@@ -235,6 +235,26 @@ def get_file_permission(self, node: str, path: str) -> dict:
...
+        Args:
+            node (str): The node on which command has to execute
+            fqpath (str): The fully-qualified path to the file.
+            perms (str): A permissions string as passed to chmod.
+
+        Returns:
+            True on success. False on fail.
+        """
+        cmd = f"chmod {perms} {fqpath}"
+        ret = self.execute_abstract_op_node(cmd, node)
```
Comment (nik-redhat) [inline common/ops/support_ops/io_ops.py:250]: In such ops, better to pass, `False` to except flag by default.

File: tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        redant.volume_start(self.vol_name, self.server_list[0])
+
+        # Wait for all volume processes to be up and running.
+        if not (redant.
+                wait_for_volume_process_to_be_online(self.vol_name,
+                                                     self.server_list[0],
+                                                     self.server_list)):
+            raise Exception("All volume processes are not up")
+        # Adding sleep for the mount to be recognized by client.
+        sleep(3)
```
Comment (nik-redhat) [inline tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py:72]: is this sleep necessary?

**Review by nik-redhat — DISMISSED** [review]

File: tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py (hunk 2 above)
Comment (srijan-sivakumar) [reply tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py:72]: I think that was for gluster to recognize the change..going by the comment 😆

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py:72]: It was in the tc so i left it there

File: common/ops/support_ops/io_ops.py (hunk 1 above)
Comment (aujjwal-redhat) [reply common/ops/support_ops/io_ops.py:250]: Ack

File: tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py (hunk 2 above)
Comment (srijan-sivakumar) [reply tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py:72]: re baba...aise nahi karte 😛 ...do analyze its utility before migrating..

Comment (aujjwal-redhat) [reply tests/functional/glusterd/test_mountpoint_ownership_post_volume_restart.py:72]: :laughing: ye beraham duniya, sone bhi nahi deti 3 sec

**Comment by aujjwal-redhat** [comment]
```js
 vol_destroy_test-dist-disp Succeeded
Summary:
+---------------+-------+--------------+
|    Category   | Cases | Pass Percent |
+---------------+-------+--------------+
| nonDisruptive |   1   |    100.0     |
|   Disruptive  |   0   |      0       |
|     Total     |   1   |    100.0     |
+---------------+-------+--------------+
Table:
 vol_create_test
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     rep     |     PASS    | 8.040963172912598  |
|     dist    |     PASS    | 6.068718433380127  |
|     disp    |     PASS    |  8.79183316230774  |
|   dist-rep  |     PASS    | 9.433831453323364  |
|   dist-arb  |     PASS    | 9.338790893554688  |
|  dist-disp  |     PASS    | 12.542724370956421 |
+-------------+-------------+--------------------+

 test_mountpoint_ownership_post_volume_restart
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     rep     |     PASS    | 21.46348285675049  |
|     dist    |     PASS    | 21.115946531295776 |
|     disp    |     PASS    | 26.209532260894775 |
|   dist-rep  |     PASS    | 25.855713844299316 |
|   dist-arb  |     PASS    | 24.591617107391357 |
|  dist-disp  |     PASS    | 34.84991431236267  |
+-------------+-------------+--------------------+

 vol_destroy_test
 
+-------------+-------------+--------------------+
| Volume Type | Test Result |  Time taken (sec)  |
+-------------+-------------+--------------------+
|     rep     |     PASS    | 8.858366250991821  |
|     dist    |     PASS    | 10.305217266082764 |
|     disp    |     PASS    | 13.100492715835571 |
|   dist-rep  |     PASS    | 12.073570251464844 |
|   dist-arb  |     PASS    | 13.29908013343811  |
|  dist-disp  |     PASS    | 20.26918315887451  |
+-------------+-------------+--------------------+



Framework runtime : 298.12758708000183
```

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**